### ISSUE TITLE
Admin UI UX improvements needed

### ISSUE THREAD

**Issue by jvanulde** [fgpv-vpgf/rcs#75]
Please provide some examples of usage and hints on the UI so that users know how to use it.

**Comment by jvanulde** [comment]
Also, if the version of RCS can be provided/displayed that would be helpful.

**Comment by dan-bowerman** [comment]
Hi Joost,

We will be cleaning up the test page with working examples soon. A number of the services in those examples are no longer available, so we want to resolve this. Please note, this page was never intended to be used as an official administrative interface, rather it was meant for testing various use cases when code changes were made.

Newer version of the RCS (I think yours is older, it does not appear to have the endpoint) can access the version # via a URL such as this: http://section917.cloudapp.net:8000/v2/version.

**Comment by jvanulde** [comment]
Indeed, it's older. However, I get a 500 error when trying ~/v2/version on v2.3.1. Might be an installation issue though...
